src/azure-cli/azure/cli/command_modules/sql/custom.py

```diff
@@ -425,6 +425,19 @@ class FailoverPolicyType(Enum):
     manual = 'Manual'
 
 
+class SqlServerMinimalTlsVersionType(Enum):
```

Comment:
You should have defined this enum in swagger so that it showed up in generated sdk. Then you wouldn't need to define it here. This is what I was asking about on your swagger review. :)

Reply:
Yes, sorry about that, we have plans to fix it and do it the right way :)

Reply:
Thank you, please do :)